Find a functionally equivalent code:android.graphics.Canvas.drawPoint:COMMENT
<android.graphics.Canvas: void drawPoint(float,float,Paint)>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Helper for drawPoints() for drawing a single point.
 */

@@ -1 +1,3 @@
-
+{
+    native_drawPoint(mNativeCanvasWrapper, x, y, paint.mNativePaint);
+}

